src/styles.css

```
@@ -1,407 +1,407 @@
+}
+
+/* ============================================
+   react-day-picker Animation (Calendar)
+   ============================================ */
+
+.rdp-root {
+    --rdp-animation_duration: 0.20s;
+    --rdp-animation_timing: cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+@keyframes rdp-slide_in_left {
```

Stylelint (17.11.0)

[error] Expected keyframe name "rdp-slide_in_left" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)